string filename
---
bool status
string message
string coord_frame 
float32[] position_array 
